app/assets/stylesheets/modules/_uploads.styl

```diff
@@ -155,3 +154,10 @@
 
     .upload-viewer-button
       margin 10px
+
+.view-file-details
+  border 1px solid #ddd
+
+.row-details
```

Comment:
This rule should be scoped to children of `view-file-details`. It's a class name that is pretty general, so we want to make sure it doesn't collide with the same name in some other file if it's a top-level rule.

Reply:
@ragesoss sorry I didn't get you. So what should I change the name of `view-file-details` to?

Reply:
That name is fine. You should change the scope of `.row-details` so that it's a rule that only applies to children of `view-file-details` elements. ie, indent that rule so that it's a sub-rule of `view-file-details`.